crates/sdk/tests/two-paras-same-id.rs

```diff
@@ -0,0 +1,44 @@
+use zombienet_sdk::{NetworkConfigBuilder, NetworkConfigExt};
+
+#[tokio::test(flavor = "multi_thread")]
+async fn two_paras_same_id() {
+    tracing_subscriber::fmt::init();
+    let network = NetworkConfigBuilder::new()
+        .with_relaychain(|r| {
+            r.with_chain("rococo-local")
+                .with_default_command("polkadot")
+                .with_default_image("docker.io/parity/polkadot:v1.7.0")
+                .with_node(|node| node.with_name("alice"))
+                .with_node(|node| node.with_name("bob"))
+        })
+        .with_parachain(|p| {
+            p.with_id(2000)
+                .with_default_command("polkadot-parachain")
+                .with_default_image("docker.io/parity/polkadot-parachain:1.7.0")
+                .with_collator(|n| n.with_name("collator"))
+        })
+        .with_parachain(|p| {
+            p.with_id(2000)
+                .with_default_command("polkadot-parachain")
+                .with_default_image("docker.io/parity/polkadot-parachain:1.7.0")
+                .with_registration_strategy(zombienet_sdk::RegistrationStrategy::Manual)
+                .with_collator(|n| n.with_name("collator1"))
+        })
+        .build()
+        .unwrap()
+        .spawn_native()
```

Comment:
Let's set the provider using _env vars_ like we use [here](https://github.com/paritytech/zombienet-sdk/blob/main/crates/sdk/tests/smoke-native.rs#L24-L28)

Reply:
good point 👍 done!